importer/Uploader.py

```diff
@@ -220,16 +223,17 @@ def add_claims(self, wd_item, claims, log):
                                     quals[qual], qual)
                                 qualifier = self.wdstuff.Qualifier(qual, value)
                                 wd_value.addQualifier(qualifier)
-                        for ref in refs:
-                            """
-                            This only works if it's a url.
-                            If we have references of different sort,
-                            this will have to be appended.
-                            """
-                            if utils.is_valid_url(ref):
-                                ref = self.make_url_reference(ref)
-                            else:
-                                ref = self.make_stated_in_reference(ref)
+                        if refs:
```

Comment:
The alternative would be to set `refs = []` on [L119](https://github.com/Vesihiisi/COH-tools/commit/669e78b0fb0044e1abc4656aefccc74fb59abde2#diff-3c84fc200ee0223310cef2e0937815beR119). That wouldn't require any other changes and might actually be clearer (since `refs=None` before that line is what triggers default ref)